app/styles/u.css

```diff
@@ -147,11 +147,6 @@
   font-weight: var(--bold);
 }
 
-.u__results-main {
-  max-width: 52rem;
-  margin: 0 auto;
-}
```

Comment:
My first idea was to change this to:

```css
My first idea was to change this to:

```css
.u__results-main {
   width: 100%;
   max-width: 52rem;
   margin: 0 auto;
 }
```

But then I realised that the user profile page just uses the `.main` class defined in `layout.css`, which has this line and also a padding that made the table's div not touch the window edges on small viewports.

One difference is that `.main` uses `48rem` as a max-width (including the padding i believe).
Let me know if that class shouldn't be used here / If the 52rem were purposefully picked.

Reply:
I don't remember if there was some special meaning but just testing your PR this looks great so let's go with this 👍